### DISCUSSION TITLE
Happy New Year 2022! 🎉 🧨 💥

### DISCUSSION THREAD

**Discussion by MarcinOrlowski** [logisim-evolution/logisim-evolution#1344] [General]
Happy New Year 2022! 🎉 🧨 💥 

PS: 

![output](https://user-images.githubusercontent.com/8041294/147796811-787f86fe-39c1-47cf-ae2c-afafc38397c2.gif)

**Comment by maehne** [comment]
@MarcinOrlowski: Thank you!

I wish the same to all our users and contributors!

PS @MarcinOrlowski: I look forward to your PR with the filter feature! It looks really nice.

**Comment by davidhutchens** [comment]
Happy New Year everyone!

@MarcinOrlowski: The filter looks nifty!
